README.md

```diff
@@ -6,82 +6,82 @@ OpenShift Application Platform
-* Team and user isolation of containers, builds, and network communication in an easy multi-tenant system
-  * Allow developers to run containers securely with fine-grained controls in production
-  * Limit, track, and manage the developers and teams on the platform
-* Integrated Docker registry, automatic edge load balancing, cluster logging, and integrated metrics
+**For developers:**
+
+* Easily build applications with integrated service discovery and persistent storage.
+* Quickly and easily scale applications to handle periods of increased demand.
+  * Support for automatic high availability, load balancing, health checking, and failover.
+* Push source code to your Git repository and automatically deploy containerized applications.
+* Web console and command-line client for building and monitoring applications.
+
+**For system administrators:**
+
+* Centralized administration and management of an entire stack, team, or organization.
+  * Create reusable templates for components of your system, and iteratively deploy them over time.
+  * Roll out modifications to software stacks to your entire organization in a controlled fashion.
+  * Integration with your existing authentication mechanisms, including LDAP, Active Directory, and public OAuth providers such as GitHub.
+* Multi-tenancy support, including team and user isolation of containers, builds, and network communication.
+  * Allow developers to run containers securely with fine-grained controls in production.
+  * Limit, track, and manage the developers and teams on the platform.
+* Integrated Docker registry, automatic edge load balancing, cluster logging, and integrated metrics.
 
 **Learn More:**
 
-* **[Public Documentation](http://docs.openshift.org/latest/welcome/index.html)**
-* The **[Trello Roadmap](https://ci.openshift.redhat.com/roadmap_overview.html)** covers the epics and stories being worked on (click through to individual items)
+* **[Public Documentation](https://docs.openshift.org/latest/welcome/)**
+  * **[API Documentation](https://docs.openshift.org/latest/rest_api/openshift_v1.html)**
 * **[Technical Architecture Presentation](https://docs.google.com/presentation/d/1Isp5UeQZTo3gh6e59FMYmMs_V9QIQeBelmbyHIJ1H_g/pub?start=false&loop=false&delayms=3000)**
 * **[System Architecture](https://github.com/openshift/openshift-pep/blob/master/openshift-pep-013-openshift-3.md)** design document
-* **[API Documentation](http://docs.openshift.org/latest/rest_api/openshift_v1.html)**
+* The **[Trello Roadmap](https://ci.openshift.redhat.com/roadmap_overview.html)** covers the epics and stories being worked on (click through to individual items).
 
 For questions or feedback, reach us on [IRC on #openshift-dev](https://botbot.me/freenode/openshift-dev/) on Freenode or post to our [mailing list](https://lists.openshift.redhat.com/openshiftmm/listinfo/dev).
 
-
 Getting Started
 ---------------
 
 ### Installation
```

Comment:
It'll be added when I rev to 1:3.0-alpha.2

On Jun 20, 2016, at 7:34 PM, Jonathan Yu <notifications@github.com> wrote:

In README.md
<https://github.com/openshift/origin/pull/8007#discussion_r67785927>:

>
>  For questions or feedback, reach us on [IRC on #openshift-dev](https://botbot.me/freenode/openshift-dev/) on Freenode or post to our [mailing list](https://lists.openshift.redhat.com/openshiftmm/listinfo/dev).
>
> -
>  Getting Started
>  ---------------
>
>  ### Installation

We don't mention the oc cluster up method yet, should we add it? It's not
one of the documented installation options
<https://docs.openshift.org/latest/getting_started/administrators.html>,
either. From what little I understand of it, this might be the easiest way
to get an OpenShift installation running for development purposes, since it
only requires docker and a copy of the oc binary.

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/8007/files/bc4da8fa99b5d01b492ccd85bada33fcb9cd33dd#r67785927>,
or mute the thread
<https://github.com/notifications/unsubscribe/ABG_p_sq7BN1UCmSrzyXc9vP7K3465dtks5qNyOVgaJpZM4HwhAG>
.